server/utils/jsonApi.js

```diff
@@ -76,6 +76,9 @@ const getCategoryId = categories => {
   };
 };
 
+const isUnpublished = ({ unpublishOn = null }) =>
+  !unpublishOn || unpublishOn >= new Date().getTime();
```

Comment:
Have we checked how this handles timezones?
The unpublished date should always be in UTC, so as long as `new Date().getTime()` is also UTC we should be OK.

Reply:
getTime return UTC - so this should be fine 👍  Its also all server side.